packages/grid/_modules_/grid/components/GridCell.tsx

```diff
@@ -102,6 +101,21 @@ export const GridCell: React.FC<GridCellProps> = React.memo((props) => {
     [publish, publishClick],
   );
 
+  const onDragEnter = React.useCallback((event) => apiRef!.current.onColItemDragEnter(event), [
+    apiRef,
+  ]);
+
+  const onDragOver = React.useCallback(
```

Comment:
We have a convention that we use `handle` instead of `on` so this method will become `handleDragOver`.

```suggestion
  const handleDragOver = React.useCallback(
```

Reply:
I'll rename it. I took these names from [here](https://github.com/mui-org/material-ui-x/blob/master/packages/grid/_modules_/grid/components/columnHeaders/GridColumnHeaderItem.tsx#L74).